homeassistant/components/owlet/sensor.py

```diff
@@ -0,0 +1,151 @@
+"""Support for Owlet sensors."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    UnitOfTemperature,
+    UnitOfTime,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from .const import DOMAIN, SLEEP_STATES
+from .coordinator import OwletCoordinator
+from .entity import OwletBaseEntity
+
+
+@dataclass
+class OwletSensorEntityDescription(SensorEntityDescription):
+    """Represent the owlet sensor entity description."""
+
```

Comment:
```suggestion
    available_during_charging: bool

```

We should set it via the `OwletSensorEntityDescription` so it's easier when we add new sensors